scripts/replaceStrings.ts

```diff
@@ -0,0 +1,30 @@
+/* eslint @typescript-eslint/no-var-requires: 0 */
```

Comment:
The name of this file is a bit confusing. Maybe we should rename it to something like replacePakcageVersion 
setPackageVersion

Reply:
updated file name